src/pages/content-edit/head.css

```diff
@@ -127,3 +127,21 @@
 .end {
     flex: 1 0 50%;
 }
+
+.invalidDate {
+    border: thin solid #e61010;
+    color: #e61010;
+}
+
+.clearSchedule {
+    background: transparent;
+    border: none;
+    color: grey;
+    font-style: italic;
+}
+.clearSchedule:hover {
```

Comment:
These should match the formatting in the rest of the file and have an extra newline between each.